### DISCUSSION TITLE
In browser mode, it seems that the iframe is destroyed every time after executing the test.

### DISCUSSION THREAD

**Discussion by njzydark** [vitest-dev/vitest#6202] [Q&A] [answered]
Can we keep the iframe so that I can inspect the component's UI?

I am using @testing-library/react. I did not manually call cleanup, but the DOM is indeed being destroyed. According to the official documentation, it seems to automatically clean up, and I couldn't find a manual way to close it.

**Comment by sheremet-va** [comment]
You need to set `PTL_SKIP_AUTO_CLEANUP` to `true`: https://testing-library.com/docs/preact-testing-library/api/#cleanup

```
PTL_SKIP_AUTO_CLEANUP=true vitest --browser
```

**Reply by njzydark** [reply]
> You need to set `PTL_SKIP_AUTO_CLEANUP` to `true`: https://testing-library.com/docs/preact-testing-library/api/#cleanup
> 
> ```
> PTL_SKIP_AUTO_CLEANUP=true vitest --browser
> ```

Setting this environment variable didn't work. I found that as long as I enable `globals` configuration, the DOM would be destroyed after the test execution, and the DOM would be retained only if I disabled the `globals` configuration.

**Reply by njzydark** [reply]
I am now heavily reliant on enabling the `globals` configuration because I am using vitest in a `rush monorepo`. To avoid installing vitest dependencies in every package, I have created a separate package, such as `my-vitest`. Therefore, I import test files from this package, and if I want to use the `vi.mock` method, I must enable the `globals` configuration.

**Reply by sheremet-va** [reply, marked as answer]
Sorry, I linked you the preact name, you need to use `RTL_SKIP_AUTO_CLEANUP` instead
